ibis/expr/types/relations.py

```diff
@@ -2551,37 +2567,36 @@ def cross_join(
         >>> agg = t.drop("year").agg(s.across(s.numeric(), _.mean()))
         >>> expr = t.cross_join(agg)
         >>> expr
-        ┏━━━━━━━━━┳━━━━━━━━━━━┳━━━━━━━━━━━━━━━━━━┳━━━━━━━━━━━━━━━━━┳━━━┓
-        ┃ species ┃ island    ┃ bill_length_mm_x ┃ bill_depth_mm_x ┃ … ┃
-        ┡━━━━━━━━━╇━━━━━━━━━━━╇━━━━━━━━━━━━━━━━━━╇━━━━━━━━━━━━━━━━━╇━━━┩
-        │ string  │ string    │ float64          │ float64         │ … │
-        ├─────────┼───────────┼──────────────────┼─────────────────┼───┤
-        │ Adelie  │ Torgersen │             39.1 │            18.7 │ … │
-        │ Adelie  │ Torgersen │             39.5 │            17.4 │ … │
-        │ Adelie  │ Torgersen │             40.3 │            18.0 │ … │
-        │ Adelie  │ Torgersen │              nan │             nan │ … │
-        │ Adelie  │ Torgersen │             36.7 │            19.3 │ … │
-        │ Adelie  │ Torgersen │             39.3 │            20.6 │ … │
-        │ Adelie  │ Torgersen │             38.9 │            17.8 │ … │
-        │ Adelie  │ Torgersen │             39.2 │            19.6 │ … │
-        │ Adelie  │ Torgersen │             34.1 │            18.1 │ … │
-        │ Adelie  │ Torgersen │             42.0 │            20.2 │ … │
-        │ …       │ …         │                … │               … │ … │
-        └─────────┴───────────┴──────────────────┴─────────────────┴───┘
-        >>> from pprint import pprint
-        >>> pprint(expr.columns)
+        ┏━━━━━━━━━┳━━━━━━━━━━━┳━━━━━━━━━━━━━━━━┳━━━━━━━━━━━━━━━┳━━━━━━━━━━━━━━━━━━━┳━━━┓
+        ┃ species ┃ island    ┃ bill_length_mm ┃ bill_depth_mm ┃ flipper_length_mm ┃ … ┃
+        ┡━━━━━━━━━╇━━━━━━━━━━━╇━━━━━━━━━━━━━━━━╇━━━━━━━━━━━━━━━╇━━━━━━━━━━━━━━━━━━━╇━━━┩
+        │ string  │ string    │ float64        │ float64       │ int64             │ … │
+        ├─────────┼───────────┼────────────────┼───────────────┼───────────────────┼───┤
+        │ Adelie  │ Torgersen │           39.1 │          18.7 │               181 │ … │
+        │ Adelie  │ Torgersen │           39.5 │          17.4 │               186 │ … │
+        │ Adelie  │ Torgersen │           40.3 │          18.0 │               195 │ … │
+        │ Adelie  │ Torgersen │            nan │           nan │                 ∅ │ … │
```

Comment:
```suggestion
        │ Adelie  │ Torgersen │            nan │           nan │              NULL │ … │
```

Reply:
The changes to the null repr haven't been merged to the 6.x.x branch yet; can't make this change until that happens.